src/components/eip3643/admin/DeployToken.tsx

```diff
@@ -154,11 +195,112 @@ export default function DeployToken({ onClose = () => {} }) {
+                }}
+                value={form.values.maxOwnershipCountryCode}
+              >
+                <Stack direction="row">
+                  <Radio value={CountryCodesISO.US.toString()}>US</Radio>
+                  <Radio value={CountryCodesISO.NON_US.toString()}>
+                    Non US
+                  </Radio>
+                </Stack>
+              </RadioGroup>
+            </FormControl>
+
+            <FormControl pt={5}>
+              <FormLabel>
+                Max percentage of tokens owned by a local identity
+              </FormLabel>
+              <NumberInput
+                name="maxOwnershipCountryLocalPercentage"
+                variant="outline"
+                value={form.values.maxOwnershipCountryLocalPercentage}
+                min={0}
+                max={10000}
+                precision={0}
+                onChange={(value) =>
+                  form.setFieldValue(
+                    "maxOwnershipCountryLocalPercentage",
+                    value,
+                  )
+                }
+              >
+                <NumberInputField />
+                <NumberInputStepper>
+                  <NumberIncrementStepper />
+                  <NumberDecrementStepper />
+                </NumberInputStepper>
+              </NumberInput>
+
+              <FormHelperText>
+                Set value in basis points (e.g. 185 = 1.85%))
+              </FormHelperText>
+              <FormHelperText>
+                Current value in % is:{" "}
+                {form.values.maxOwnershipCountryLocalPercentage / 100}%
+              </FormHelperText>
+            </FormControl>
+
+            <FormControl pt={5}>
+              <FormLabel>
+                Max percentage of tokens owned by a non-local identity
+              </FormLabel>
+              <NumberInput
+                name="maxOwnershipCountryLocalPercentage"
+                variant="outline"
+                value={form.values.maxOwnershipCountryNonLocalPercentage}
+                min={0}
+                max={10000}
+                precision={0}
+                onChange={(value) =>
+                  form.setFieldValue(
+                    "maxOwnershipCountryNonLocalPercentage",
+                    value,
+                  )
+                }
+              >
+                <NumberInputField />
+                <NumberInputStepper>
+                  <NumberIncrementStepper />
+                  <NumberDecrementStepper />
+                </NumberInputStepper>
+              </NumberInput>
+
+              <FormHelperText>
+                Set value in basis points (e.g. 185 = 1.85%))
+              </FormHelperText>
+              <FormHelperText>
+                Current value in % is:{" "}
+                {form.values.maxOwnershipCountryNonLocalPercentage / 100}%
+              </FormHelperText>
+            </FormControl>
+
+            {complianceError && (
```

Comment:
I think we could apply similar result component like i did here: https://github.com/web3-nomad/accelerator-defi-EIP-ui/pull/87/files#diff-accff24613ee89854e115eda7d80e368d7a9fdd38c65324fc47f44f9bd399dafR1.

Reply:
good point, lets cover it separately after the compo gets into main